### PR TITLE
Increase the gap between the email and the text in the contact text

### PR DESCRIPTION
**Pull request by Rishav1707**
## Purpose

The purpose of this Pull Request is to fix #155 <issue-number>

### Before
206857804-2b43ac31-91f3-4793-9d3d-b6271a4bdd7a
### After
Screenshot 2022-12-11 094803

### Preview Link

https://pr-156-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Rishav1707** [comment]
Please run the workflows @YoshithaRathnayake

File: dark/index.html [hunk 1]
```diff
@@ -239,7 +239,7 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
           just below.
         </p>
         <p class="contact-text">
-          <i class="fa fa-envelope"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
```
Comment (YoshithaRathnayake) [inline dark/index.html:242]: Don't remove the `fa fa-envelope` bcs it is the email icon @Rishav1707

File: dark/index.html [hunk 2]
```diff
@@ -239,7 +239,7 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
           just below.
         </p>
         <p class="contact-text">
-          <i class="fa fa-envelope"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
+          <i class="px-1"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
```
Comment (YoshithaRathnayake) [inline dark/index.html:242]:
```suggestion
          <i class="fa fa-envelope px-1"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 1 above)
Comment (Rishav1707) [reply dark/index.html:242]: Done

**Comment by YoshithaRathnayake** [comment]
Can you update the preview link and the screenshot @Rishav1707

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Can you update the preview link and the screenshot @Rishav1707

**Comment by Rishav1707** [comment]
> Can you update the preview link and the screenshot @Rishav1707

Sure

**Comment by Rishav1707** [comment]
> Can you update the preview link and the screenshot @Rishav1707

Done @YoshithaRathnayake

**Comment by YoshithaRathnayake** [comment]
> > Can you update the preview link and the screenshot @Rishav1707
> 
> Done @YoshithaRathnayake

Nice

**Review by YoshithaRathnayake — APPROVED** [review]

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @Rishav1707

**Comment by Rishav1707** [comment]
> Thank You! @Rishav1707

Your welcome @YoshithaRathnayake

**Pull request merged**